### ISSUE TITLE
Discrepancy in Reported Fusion Accuracy and Reproducibility Concerns in Fine-Tuning Setup

### ISSUE THREAD

**Issue by rickyeiji** [kyrie-23/linear_task_arithmetic#3]
Hello, author. While working nearby, I noticed that after fine-tuning ViT/32-B on eight visual tasks, the model fusion evaluation only achieved 71%, which differs from the 78% performance reported in the original paper. Could you please confirm whether the code in this repository guarantees reproducibility? I also observed that some parameters in README.md still retain the linear-2 fine-tuning mode.

Here is the evaluation command currently shown:
```
python eval_task_addition.py \
--model=ViT-B-32 \
--finetuning-mode=bias-attn \
--n-eval-points=21
```
Would you mind verifying whether this configuration reflects the correct experimental setup used to obtain the 78% result?

**Comment by kyrie-23** [comment]
Hi, thanks for your interest. Please make sure to use the linear-2 mode for reproduction, the bias-attn is for ablation.
You can simply change line 28 to 'linear-2' once you have the checkpoints. Sorry for any confusion.
`elif args.finetuning_mode == "linear-2":`

**Comment by rickyeiji** [comment]
Hi, Thanks for your reply! I have a question about finetuning by linear-2. Here is finetune.py, I don't know whether 'linear-2' mode let linearized_finetuning = True?

```
    assert args.finetuning_mode in [
        "linear",
        "standard",
        "bias-attn",
        "linear-2",
    ], "Only linear and standard fine-tuning are supported."

    linearized_finetuning = args.finetuning_mode == "linear"
    if linearized_finetuning:
        print("Using linearized fine-tuning.")
```


Could you update the newest code? linear-2 is not in finetune.py which can be identified by assert args.finetuning_mode in ...

**Comment by kyrie-23** [comment]
Hi, we have updated the code, please simply try the 'linear-2' mode.
